back/src/messages/input-processor.ts

```diff
@@ -96,12 +94,14 @@ const handleEstablishConnectionTrainer = async (socketInfo: SocketInfo, room: st
     return [];
   }
 
+  const roomAvailable: boolean = await isRoomAvailable(room);
+
+  if(roomAvailable) {
+    saveRoomInfo({room, content:""}, "");
+  }
+
   if (await isRoomAvailable(room) || !(await isExistingConnection(socketInfo.connectionId))) {
```

Comment:
Why don't you use here `roomAvailable` constant?

Reply:
It should be in that way as in the handleEstablishConnectionStudent function, an oblivion. Will be fixed in the next push.

Reply:
Done and pushed